assets/src/containers/IndexPage.js

```diff
@@ -24,6 +26,9 @@ const styles = theme => ({
 function IndexPage (props) {
   const { classes, courseInfo, courseId } = props
 
+  if (isObjectEmpty(courseInfo.course_view_options))
+    return (<Error>No data visualizations have been added for this course.</Error>)
```

Comment:
the `course_view_options` is not empty when all the views are disabled. This is how the part of Json looks like `"course_view_options": {"fa": 0, "ap": 0, "gd": 0}}`. This info is pulled from the DB.  The code never enter this case
There is error case(when DB look up failed) the Json might looks like this`"course_view_options": ""` 
I think it needs to support both the cases.

Reply:
Ah good point, okay, I'll fix this.